src/components/Header/Header.tsx

```diff
@@ -3,6 +3,7 @@ import { Button } from "components/Button/Button";
 import { Link, LinkProps } from "components/Link/Link";
 import { useOutsideClick } from "hooks/useOutsideClick";
 import { useWindowDimensions } from "hooks/useWindowDimensions";
+import { Icon } from "components/Icon/Icon";
```

Comment:
Might need to use a relative import here

Reply:
done and done.